tool.ts

```diff
@@ -260,14 +267,38 @@ export async function downloadTool(
             file.on('error', (err) => {
                 file.end();
                 reject(err);
-            })
+            });
         }
         catch (error) {
             reject(error);
         }
     });
 }
 
+function _getContentLengthOfDownloadedFile(contentLengthHeader: string | undefined): number {
```

Comment:
Please add jsdoc comment for new functions

Reply:
Fixed